services/bonita-commons/src/main/java/org/bonitasoft/engine/commons/ClassReflector.java

```diff
@@ -113,8 +113,8 @@
     public static void invokeSetter(final Object entity, final String setterName, final Class<?> parameterType, final Object parameterValue)
             throws SReflectException {
         try {
-            final Method setter = getMethod(entity.getClass(), setterName, new Class[] { parameterType });
-            setter.invoke(entity, new Object[] { parameterValue });
+            final Method setter = getMethod(entity.getClass(), setterName, parameterType);
```

Comment:
nop, however all BDM tests are working, My IDE told me it was not usefull
However if you think it might cause issues I can remove this part which is not the main purpose of the PR